.changeset/funny-points-cover.md

```diff
@@ -0,0 +1,5 @@
+---
+"@atproto/xrpc-server": minor
+---
+
+Allow upload of payload of zero bytes (e.g. empty txt file)
```

Comment:
I think this has been supported since https://github.com/bluesky-social/atproto/pull/2599

I don't see any material changes to xrpc-server here, so I think we can nix this changeset.